examples/data_pipeline/transforms.py

```diff
@@ -24,11 +24,11 @@ def forward(self, tokens: List[str]) -> List[int]:
     def insert_token(self, token: str, index: int) -> None:
         self.vocab.insert_token(token, index)
 
-    def to_ivalue(self):
-        if hasattr(self.vocab, 'to_ivalue'):
+    def __prepare_scriptable__(self):
+        if hasattr(self.vocab, '__prepare_scriptable__'):
             sp_model = self.sp_model
             new_module = PretrainedSPVocab(sp_model)
-            new_module.vocab = self.vocab.to_ivalue()
+            new_module.vocab = self.vocab.__prepare_scriptable__()
```

Comment:
Isn't prepare_scriptable recursive to members of modules? Will we even need to call this manually? Do we even need this method at all?

Reply:
This is not necessary any more since the `__prepare_scriptable__` func recursively prepares the components.